source/includes/markdown/_14-publish-your-app.html.md

```diff
@@ -68,28 +68,49 @@ The explorer is not meant to be an exhaustive tool with every endpoint in the AP
 <section>
 # Submit your App
 
-<span class="description">
-Have you built a web application that you want to share with the Asana community? Submit [this form](https://www.asana.com/apps/submit) to get your app listed in the [Asana apps directory](https://www.asana.com/apps).
-</span>
+If you've built an application that you wish to share with the Asana community, you can visit the
+[developer console](https://app.asana.com/0/my-apps) to begin the app submission process. 
+
+By submitting an app for review, you'll have the opportunity to distribute your app publicly by listing
+it in the [app directory](https://asana.com/apps) (as well as in the in-product [app gallery](/docs/installing-an-app)
```

Comment:
`/docs/installing-an-app` looks like a dead link I tried to go to `/docs/installing-an-app` but it didn't take me anywhere

Reply:
`/docs/installation-flow` is the correct one